saturn-datepicker/src/datepicker/calendar-body.ts

```diff
@@ -169,6 +185,41 @@ export class SatCalendarBody implements OnChanges {
     return date > <number>this.begin && date < <number>this.end;
   }
 
+  /** Whenever to mark cell as semi-selected before the second date is selected (between the begin cell and the hovered cell). */
+  _isBetweenOverAndBegin(date: number): boolean {
+    if (!this._cellOver) {
+      return false;
+    }
+    if (this.isBeforeSelected && !this.begin) {
+      return date > this._cellOver;
+    }
+    if (this._cellOver > this.begin) {
+      return date > this.begin && date < this._cellOver;
+    }
+    if (this._cellOver < this.begin) {
+      return date < this.begin && date > this._cellOver;
+    }
+    return false;
+  }
+
+  /** Whenever to mark cell as begin of the range. */
+  _isBegin(date: number): boolean {
+    if (this.isBeforeSelected && !this.begin) {
+      return (this._cellOver === date);
```

Comment:
Please remove extra braces

Reply:
Which braces? Those around `return (this._cellOver === date);`?